src/vendors/Datadog.Trace.Vendoring/DatadogLogging.cs

```diff
@@ -0,0 +1,126 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using Datadog.Trace.Vendoring.Serilog;
+using Datadog.Trace.Vendoring.Serilog.Events;
+using Datadog.Trace.Vendoring.Serilog.Sinks.File;
+
+namespace Datadog.Trace.Vendoring
+{
+    /// <summary>
+    /// Class for Datadog managed file logging.
+    /// </summary>
+    public static class DatadogLogging
+    {
+        private const string WindowsDefaultDirectory = "C:\\ProgramData\\Datadog .NET Tracer\\logs\\";
```

Comment:
We should use `System.Environment.GetFolderPath(System.Environment.SpecialFolder.CommonApplicationData)` to determine the path and `Path.Combine()` to combine them.
```suggestion
        private const string WindowsDefaultDirectory = Path.Combine(Environment.GetFolderPath(System.Environment.SpecialFolder.CommonApplicationData), @"Datadog .NET Tracer", "logs");
```

Reply:
But then it can't be const! :P
Yeah, sounds good.

Reply:
Indeed, my suggestion doesn't even compile! 🤦‍♂